src/domain/tokens/types.ts

```diff
@@ -4,46 +4,68 @@ export type Token = {
   address: string;
   decimals: number;
   imageUrl: string;
-  isStable?: boolean;
-  isNative?: boolean;
-  isShortable?: boolean;
+  isStable: boolean;
   name: string;
   symbol: string;
+  isTempHidden?: boolean;
+  isWrapped?: boolean;
+  baseSymbol?: string;
 };
```

Comment:
Could you add a line break here? :)

Reply:
i guess better to move such rules to prettier config or at least formalize it somehow? otherwise it's very subjective

i think it's good to add some newlines for readability, but unless we don't have such rule better to not block PRs with it imo